cont/LuaUI/rml_setup.lua

```diff
@@ -0,0 +1,27 @@
+--  file:    rml.lua
+--  brief:   RmlUi Setup
+--  author:  lov + ChrisFloofyKitsune
+--
+--  Copyright (C) 2024.
+--  Licensed under the terms of the GNU GPL, v2 or later.
+
+if (RmlGuard or not RmlUi) then
+	return
+end
+RmlGuard = true
+
+-- Load fonts
+RmlUi.LoadFontFace("Fonts/FreeMonoBold.ttf", true)
```

Comment:
what happens if a game doesn't do this?

Reply:
I honestly do not know for this one, I'll try commenting it out and get back to you.

Reply:
According to https://mikke89.github.io/RmlUiDoc/pages/cpp_manual/fonts.html :

> TrueType and OpenType fonts can be loaded into RmlUi by the application. RmlUi has no default font, so at least one font must be loaded before text can be rendered.

I have just tried commenting out the font loading in my own code, and:

- Text is no longer rendered.
- RmlUi logs warnings and complains about no font face being defined.
- Buttons and the rest of the UI works, at least in my own project.
- If you are running with the debugger (`Rml::Debugger::Initialise()`) , RmlUi loads its own "rmlui-debugger-font" to show debug text in the debug UI element inside your application window.

RmlUi describes "fallback fonts" in https://mikke89.github.io/RmlUiDoc/pages/cpp_manual/fonts.html . So if the RmlUi integration code defines at least one fallback font, unless fonts can be unloaded, text will as far as I can tell always be rendered (except for the characters that the fallback font does not include). You can also define multiple fallback fonts.